wlp-managed-8.5/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -576,10 +707,23 @@ private Element createApplication(Document doc, String deploymentName, String ar
          application.appendChild(sharedLib);
       }
 
+      if(containerConfiguration.getSecurityConfiguration() != null) {
```

Comment:
I guess you'll already have sample configurations that you're using; would you be able to add a few lines to our documentation at https://docs.jboss.org/author/display/ARQ/WLP+V8.5+-+Managed to describe these new options and help others get a head start on how they might want to use them?